compiler/control/OMROptions.hpp

```diff
@@ -1340,10 +1340,159 @@ class OMR_EXTENSIBLE Options
+      _addressToEnumerate = 0;
+      _debugEnableFlags = 0;
+      _optLevelDowngraded = false;
+      _compileExcludedmethods = false;
+      _maxUnloadedAddressRanges = 0;
+      _maxStaticPICSlots = 0;
+      _hotMaxStaticPICSlots = 0;
+      _newAotrtDebugLevel = 0;
+      _disableDLTBytecodeIndex = 0;
+      _enableDLTBytecodeIndex = 0;
+      _dltOptLevel = 0;
+      _profilingCount = 0;
+      _profilingFrequency = 0;
+      _counterBucketGranularity = 0;
+      _minCounterFidelity = 0;
+      _debugCounterWarmupSeconds = 0;
+      _insertDebuggingCounters = 0;
+      _inlineCntrCalleeTooBigBucketSize = 0;
+      _inlineCntrColdAndNotTinyBucketSize = 0;
+      _inlineCntrWarmCalleeTooBigBucketSize = 0;
+      _inlineCntrRanOutOfBudgetBucketSize = 0;
+      _inlineCntrCalleeTooDeepBucketSize = 0;
+      _inlineCntrWarmCallerHasTooManyNodesBucketSize = 0;
+      _inlineCntrWarmCalleeHasTooManyNodesBucketSize = 0;
+      _inlineCntrDepthExceededBucketSize = 0;
+      _inlineCntrAllBucketSize = 0;
+      _maxInlinedCalls = 0;
+      _dumbInlinerBytecodeSizeMaxCutoff = 0;
+      _dumbInlinerBytecodeSizeMinCutoff = 0;
+      _dumbInlinerBytecodeSizeCutoff = 0;
+      _dumbInlinerBytecodeSizeDivisor = 0;
+      _trivialInlinerMaxSize = 0;
+      _inlinerArgumentHeuristicFractionUpToWarm = 0;
+      _inlinerArgumentHeuristicFractionBeyondWarm = 0;
+      _inlinerVeryColdBorderFrequencyAtCold = 0;
+      _inlinerBorderFrequency = 0;
+      _inlinerColdBorderFrequency = 0;
+      _inlinerVeryColdBorderFrequency = 0;
+      _inlinerCGBorderFrequency = 0;
+      _inlinerCGColdBorderFrequency = 0;
+      _inlinerCGVeryColdBorderFrequency = 0;
+      _alwaysWorthInliningThreshold = 0;
+      _initialSCount = 0;
+      _enableSCHintFlags = 0;
+      _insertGCRTrees = false;
+      _maxLimitedGRACandidates = 0;
+      _maxLimitedGRARegs = 0;
+      _enableGPU = 0;
+      _isAOTCompile = false;
+      _jProfilingMethodRecompThreshold = 0;
+      _jProfilingLoopRecompThreshold = 0;
+      _blockShufflingSequence = NULL;
+      _randomSeed = 0;
+      _logListForOtherCompThreads = NULL;
+      _induceOSR = NULL;
+      _bigCalleeThreshold = 0;
+      _bigCalleeThresholdForColdCallsAtWarm = 0;
+      _bigCalleeFreqCutoffAtWarm = 0;
+      _bigCalleeHotOptThreshold = 0;
+      _bigCalleeThresholdForColdCallsAtHot = 0;
+      _bigCalleeFreqCutoffAtHot = 0;
+      _bigCalleeScorchingOptThreshold = 0;
+      _inlinerVeryLargeCompiledMethodThreshold = 0);
+      _inlinerVeryLargeCompiledMethodFaninThreshold = 0;
+      _largeCompiledMethodExemptionFreqCutoff = 0;
+      _maxSzForVPInliningWarm = 0;
+      _loopyAsyncCheckInsertionMaxEntryFreq = 0;
+      _objectFileName = 0;
+
+      memset(_options, 0, sizeof(_options));
+      memset(_disabledOptimizations, false, sizeof(_disabledOptimizations));
+      memset(_traceOptimizations, false, sizeof(_traceOptimizations));
+   }
+
+   Options()
       {
+         self()->init();
       }
```

Comment:
You're gonna have to move the body to a `.cpp` file as the compiler doesn't know how to resolve the type of `self()` at this point.